### PR TITLE
Appveyor config for windows build+installer

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: nsis/win-build [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 130aa226bd4f2c20992b38c7532978360425947f
```
Comment (arkpar) [inline nsis/win-build]: What is this about?

File: .gitmodules [hunk 2]
```diff
@@ -2,3 +2,6 @@
 	path = ethcore/res/ethereum/tests
 	url = https://github.com/ethereum/tests.git
 	branch = develop
+[submodule "nsis/win-build"]
```
Comment (arkpar) [inline .gitmodules]: What's the need for a submodule?

Comment (NikVolf) [inline .gitmodules]: so we don't put nsis plugin dll files in the parity repo?

Comment (arkpar) [inline .gitmodules]: The plugin should be just installed on the build machine along with nsis, no need to keep it in the repo

Comment (NikVolf) [inline .gitmodules]: NSIS is preinstalled with appveyor, but without plugin
submodule is just a way to install plugin

Comment (arkpar) [inline .gitmodules]: Submodule is not the best way to do it. We don't to push a dll to all users who do a recursive clone.
why not just add something like 
`git archive --remote=https://github.com/ethcore/win-build.git HEAD SimpleFC.dll`
to `appveyor.yml`?

**Comment by NikVolf** [comment]
@arkpar updated

**Pull request merged**